### ISSUE TITLE
Store counts/statistics in the database and put in place a procedure to update them

### ISSUE THREAD

**Issue by jvoight** [LMFDB/lmfdb#1236]
I think we could easily put in place procedures to have counts automatically updated every night, but we don't necessarily need this in place for the launch.  (So we'll want a simple script that can be run when someone updates the data, but is always run at night.)

**Comment by AndrewVSutherland** [comment]
The first step is to have a generic mechanism for computing and storing counts; I am going to work on this and implement it in genus 2 curves.

Once this is in place, I don't think we need to do it nightly, at least not on the production server, we only need to do it when the database changes (which is infrequent).

**Comment by AndrewVSutherland** [comment]
I'll also note that, aside from a few issues in ecnf and number fields, this issue is the main thing that is keeping me from closing #1247.

**Comment by JohnCremona** [comment]
Quite a few collections / sections do this, with some version of an update_stats() script in the scripts directory which can be easily run after any data updates.  Quite possibly, all places where we display counts have such scripts in place.  If that were true then this could be closed.

**Comment by edgarcosta** [comment]
> Quite possibly, all places where we display counts have such scripts in place.

I think that is the case, and I think we can close this. 

ps: If this is not the case, someone will tumble up on this issue again, and raise a new issue or email us.
